www/src/components/CodeBlock.jsx

```diff
@@ -6,6 +6,7 @@ import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import * as ParagonReact from '~paragon-react';
 import { Add, AddCircle } from '~paragon-icons';
 import * as FontAwesome from '@fortawesome/free-solid-svg-icons'
+import MiyazakiCard from './MiyazakiCard';
```

Comment:
Let's move `MiyazakiCard` to an `example-components` directory or something

Reply:
YES good call. I also considered showing it's code in the docs?